23-Aug-2018 04:15:48.92 
 >>>>>> 
 >>>>>> Test in progress: test_C94_CheckUsersPageDisplayedCorrectly
 >>>>>>
23-Aug-2018 04:15:50.52 Logging at address: dev.rooser.co.uk
23-Aug-2018 04:15:52.81 User admin@rooser.co.uk logged in.
23-Aug-2018 04:15:55.20 >>>>>> Test execution done, tearDown

23-Aug-2018 04:15:55.72 User logged out.
.23-Aug-2018 04:15:55.79 
 >>>>>> 
 >>>>>> Test in progress: test_C95_CheckUserAdded
 >>>>>>
23-Aug-2018 04:15:57.40 Logging at address: dev.rooser.co.uk
23-Aug-2018 04:15:59.59 User admin@rooser.co.uk logged in.
E23-Aug-2018 04:16:03.29 >>>>>> Test execution done, tearDown

E23-Aug-2018 04:16:03.95 User logged out.

======================================================================
ERROR: test_C95_CheckUserAdded (SmokeSuiteUsers.SmokeTestUsers)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/afs/inf.ed.ac.uk/user/s12/s1211898/askThomas/automationRooserMsp/Tests/SmokeSuiteUsers.py", line 37, in test_C95_CheckUserAdded
    self.users_page.add_entry(first_name, last_name, email, password, role)
  File "/afs/inf.ed.ac.uk/user/s12/s1211898/askThomas/automationRooserMsp/TestPages/UsersPage.py", line 37, in add_entry
    add_dialog.add_entry_and_save(first_name, last_name, email, password, role)
  File "/afs/inf.ed.ac.uk/user/s12/s1211898/askThomas/automationRooserMsp/TestPages/TestDialogs/UsersAddDialog.py", line 30, in add_entry_and_save
    self.role.select_input(role)
  File "/afs/inf.ed.ac.uk/user/s12/s1211898/askThomas/automationRooserMsp/TestPages/TestPageObjects/Dropdown.py", line 28, in select_input
    self.arrow.click()
  File "/afs/inf.ed.ac.uk/user/s12/s1211898/askThomas/automationRooserMsp/TestPages/TestPageObjects/Dropdown.py", line 40, in arrow
    return CTAButton(self.webdriver, By.CLASS_NAME, "Select-arrow-zone", self.index)
  File "/afs/inf.ed.ac.uk/user/s12/s1211898/askThomas/automationRooserMsp/TestPages/TestPageObjects/CTAButton.py", line 53, in __init__
    self.element = self.web_driver_find.find_all(by_type, value)[index]
IndexError: list index out of range

======================================================================
ERROR: test_C95_CheckUserAdded (SmokeSuiteUsers.SmokeTestUsers)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/afs/inf.ed.ac.uk/user/s12/s1211898/askThomas/automationRooserMsp/TestPages/UsersPage.py", line 52, in remove_entry
    row = self.table.get_row_for_field_value(HeadersUsers.FirstName, name)
  File "/afs/inf.ed.ac.uk/user/s12/s1211898/askThomas/automationRooserMsp/TestPages/TestPageObjects/Table/BaseTable.py", line 76, in get_row_for_field_value
    for row in self.rows:
  File "/afs/inf.ed.ac.uk/user/s12/s1211898/askThomas/automationRooserMsp/TestPages/TestPageObjects/Table/BaseTable.py", line 50, in rows
    all_rows = self.get_raw_rows()
  File "/afs/inf.ed.ac.uk/user/s12/s1211898/askThomas/automationRooserMsp/TestPages/TestPageObjects/Table/BaseTable.py", line 26, in get_raw_rows
    return self.table_element.find_elements_by_css_selector('tr')
  File "/afs/inf.ed.ac.uk/user/s12/s1211898/askThomas/automationRooserMsp/TestPages/TestPageObjects/Table/BaseTable.py", line 31, in table_element
    return self.web_driver_find.find_all(By.CSS_SELECTOR, '.react-bs-table')[self.table_index]
IndexError: list index out of range

----------------------------------------------------------------------
Ran 2 tests in 15.145s

FAILED (errors=2)
23-Aug-2018 04:16:04.06 <<<<<<<<<<<<<<<<Test run - Smoke complete!>>>>>>>>>>>>>>>>
